elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -1017,6 +1027,110 @@ public String chooseEngineServerAlias(String keyType, Principal[] issuers, SSLEn
         }
     }
 
+    private static class LazyDelegatingKeyManager extends DelegatingKeyManager {
```

Comment:
Not a reason to delay the feature as this is private so exposure is limited but should a class such as this be a utility in the Elytron project?